### PR TITLE
Upgrade master to .NET 4.8

### PR DESCRIPTION
**Pull request by mmisol**
### Purpose

Changes included in this PR:
1. Targets .NET version for projects upgrade to v4.8.
2. Updates SKUs in app configs.
3. Updates target for nuget packages.
4. Removes StarDefinitionsCanExceedAvailableSpace switch.
5. Fix failing WPF layout-related test by using epsilon comparison.

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [x] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [x] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

**Comment by QilongTang** [comment]
Looks good as well. @DynamoDS/dynamo Notice this branch has been tested on the new VM already but we have not switched the EngOps build VM yet so that check will kept failing before BRE finish the switch

File: src/DynamoCore/packages.config [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Greg" version="1.3.7408.23353" targetFramework="net47" />
-  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
-  <package id="RestSharp" version="105.2.3" targetFramework="net47" />
+  <package id="Greg" version="1.3.7408.23353" targetFramework="net48" />
```
Comment (mjkkirschner) [inline src/DynamoCore/packages.config:3]: I guess we'll want to update package manager client to .net 4.8 as well at some point.

File: src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs [hunk 2]
```diff
@@ -532,22 +532,6 @@ public static DynamoViewModel Start(StartConfiguration startConfiguration = new
 
         protected DynamoViewModel(StartConfiguration startConfiguration)
         {
-
-            // This can be removed after this bug is fixed in .net 4.7
-            // https://developercommunity.visualstudio.com/content/problem/244615/setfinalsizemaxdiscrepancy-getting-stuck-in-an-inf.html
-            // if the key "Switch.System.Windows.Controls.Grid.StarDefinitionsCanExceedAvailableSpace" has a value true in the 
-            // dynamoCoreWpf.config file we will set the switch here before the view is created.
-            var path = this.GetType().Assembly.Location;
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs:540]: 👍

**Comment by mmisol** [comment]
Merging despite the build errors. Hopefully we will upgrade our CI really soon.

**Pull request merged**